apps/docs/src/app/core/component-docs/facets/facet-examples/custom-facet-example.component.html

```diff
@@ -0,0 +1,25 @@
+<fd-facet-group ariaLabel="Custom Facets">
+    <fd-facet type="custom" [title]="'Product Description'" class="fd-docs-custom-facet-paragraph">
```

Comment:
The reason why only `title` is this way at some places is because of incorrect a11y. If written as `title="some string"`, the tooltip 'some string' hovers on every element as `title` is also a HTML attribute and it just places the tooltip on every element in `fd-facet`. When used with the binding `[title]` it no longer considers it as a HTML tooltip attribute and doesn't show the same tooltip all over the facet elements and works as intended.

Reply:
What about renaming `title` in this case? It will be less confusing if the user wants to add a tooltip.

Reply:
sure, I will rename it as `facetTitle` instead.

Reply:
pushed in changes with `facetTitle` now, please check.